critiquebrainz/db/users.py

```diff
@@ -70,3 +73,543 @@ def get_many_by_mb_username(usernames):
             else:
                 user["avatar_url"] = gravatar_url(default_gravatar_src)
         return users
+
+
+def get_by_id(user_id):
+    """Get user from user_id
+
+    Args:
+        user_id(uuid): ID of the user
+
+    Returns:
+        Dictionary with the following structure:
+        {
+            "id": (uuid)
+            "display_name": (str)
+            "email": (str)
+            "created": (datetime)
+            "musicbrainz_id": (str)
+            "show_gravatar": (bool)
+            "is_blocked": (bool)
+        }
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT id, display_name, email, created,
+                   musicbrainz_id, show_gravatar, is_blocked
+              FROM "user"
+             WHERE id = :user_id
+        """), {
+            "user_id": user_id
+        })
+        row = result.fetchone()
+    return dict(row) if row else None
+
+
+def create(display_name, **kwargs):
+    """Create user using the given details
+
+    Args:
+        display_name(str): display_name of the user,
+        musicbrainz_id(str, optional): musicbrainz_id of user,
+        email(str, optional): email of the user,
+        show_gravatar(bool, optional): whether to show gravatar(default: false)
+        is_blocked(bool, optional): whether user is blocked(default: false)
+
+    Returns:
+        Dictionary with the following structure:
+        {
+            "id": (uuid)
+            "display_name": (str)
+            "email": (str)
+            "created": (datetime)
+            "musicbrainz_id": (str)
+            "show_gravatar": (bool)
+            "is_blocked": (bool)
+        }
+    """
+    id = str(uuid.uuid4())
```

Comment:
You can avoid making a variable and just move the `str(uuid.uuid4())` call into a dict in `sqlalchemy.text` below.